### PR TITLE
Port interpolation functionality to Animate

### LINKED ISSUE #147: Port interpolation functionality to Animate
See https://github.com/pyroteus/animate/issues/94.

### PR DESCRIPTION
**Pull request by jwallwork23**
Closes #147.

### CODE REVIEW

File: goalie/mesh_seq.py [hunk 1]
```diff
@@ -8,11 +8,12 @@
...
 from firedrake.pyplot import triplot
 from .function_data import ForwardSolutionData
-from .interpolation import transfer
 from .log import pyrint, debug, warning, info, logger, DEBUG
 from .options import AdaptParameters
+from animate.interpolation import transfer
 from animate.quality import QualityMeasure
-from .utility import AttrDict, Mesh
+from animate.utility import Mesh
+from .utility import AttrDict
```
Comment (ddundo) [inline goalie/mesh_seq.py:16]: @jwallwork23 I saw now that `.utility` is separated from other relative imports. And imports are a bit disordered in general. 

Perhaps we should (in a separate issue) organise imports manually or e.g. add [isort](https://pycqa.github.io/isort/index.html) to pre-commit?

**Review by ddundo — APPROVED** [review]
Besides the import issue on the CI side here, it looks good! I only left a minor comment that might be relevant for a new issue. Thanks again :)

**Comment by jwallwork23** [comment]
> Besides the import issue on the CI side here, it looks good! I only left a minor comment that might be relevant for a new issue. Thanks again :)

Thanks. The CI issue is because it isn't using the latest Animate after https://github.com/mesh-adaptation/animate/pull/105 was merged. I'm updating the Docker image now.

File: goalie/mesh_seq.py (hunk 1 above)
Comment (jwallwork23) [reply goalie/mesh_seq.py:16]: Good idea. See #162.

**Pull request merged**